### ISSUE TITLE
Conditional formatting should treat each column of a pivot table as a selectable column

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#11164]
When working with a one-metric, two-dimension table that I've pivoted, I'd expect to see options for each of the columns created by the pivoted dimension. E.g. in this example, I'd want options for `Doohickey`, `Gadget`, `Gizmo`, and `Widget`:

![image](https://user-images.githubusercontent.com/2223916/67031936-6e3a8d80-f0c7-11e9-8982-150d9d92d8ad.png)

The problem  with the current implementation is that it forces me to always have a single color range across the entire table. It also means I can't have a separate rule for each pivot column in order to do things like assign a different color to it.

![image](https://user-images.githubusercontent.com/2223916/67031827-33d0f080-f0c7-11e9-870e-5dfe535dbd01.png)

**Comment by tlrobinson** [comment]
I don't agree each pivoted value should be treated as a separate column in the settings, at least by default. I feel like most of the time you want the same formatting options applied to every column of a pivot table.

This is one of the things that bugs me about how we treat settings for series breakouts in line charts. If you want to change the line style for the whole chart you have to change every breakout value's settings manually. Plus if a new value is later added it won't get those settings unless you go back and manually add them.

Also, applying a "range" conditional formatting to a pivot table is how we currently allow you to do a  "heatmap", which requires using the "global" min and max values, rather than per column (though the wording in the range settings is a little ambiguous, "column" is referring to the underlying result columns rather than the pivot table visualization's columns)

**Comment by mazameli** [comment]
> I don't agree each pivoted value should be treated as a separate column in the settings, at least by default. I feel like most of the time you want the same formatting options applied to every column of a pivot table.

It really depends on if you want to range to start/stop from the lowest/highest range in the entire table vs. from the lowest/highest values in each column.

> Also, applying a "range" conditional formatting to a pivot table is how we currently allow you to do a "heatmap", which requires using the "global" min and max values, rather than per column 

You could still do this in my proposal by selecting all the columns and choosing the option that we already present when multiple columns are selected to use the smallest/largest values present in _all_ of the columns. The main thing that's missing is a select-all/select-none toggle for the column picker.

> (though the wording in the range settings is a little ambiguous, "column" is referring to the underlying result columns rather than the pivot table visualization's columns)

Yep, I understand that, and I agree it's confusing.

**Comment by rthouvenin** [comment]
Different but related: I would like to have a color range apply to rows of a pivoted table.For example in the example above, I may want to see which category got the most Facebook sources, and which got the most Google sources, and not care so much whether the number of Facebook sources is higher than Google sources.

This is especially relevant when the rows are different metrics that are not really comparable.

Should I open a separate issue for this?
